Document changes to the XDK110 platform (#236)
eclipse/mita: website/site/content/platforms/xdk110.md

### Binary Name

To customize the produced binary name configure `XDK110.applicationName: string` . The default name is `MitaApplication`:

```TypeScript
setup XDK110 {
  applicationName = "ShockDetector";
}
```

### Startup Delay

> To debug startup you can configure a startup delay so the XDK can connect via USB before setting up devices, connectivity, etc.:

Reviewer: ```suggestion
To debug the startup process you can configure a startup delay so the XDK can connect via USB before setting up devices, connectivity, etc.:
```

```TypeScript
setup XDK110 {
  startupDelay = 5000; /* wait 5 seconds before initialization */
}
```

### Extension Bus Power

> Per default the XDK doesn't power the 2.5V and 3.3V power lines on its extension bus. Some system resources like I2C or GPIO already change this, however in some cases you need to do this manually for example when using the ADC with a microphone:

Reviewer: ```suggestion
Per default the XDK doesn't power up the 2.5V and 3.3V lines on its extension bus. Some system resources like I2C or GPIO already change this, however in some cases you need to do this manually for example when using the ADC with a microphone:
```

   | `high_brightness: bool`                       | Set to `true` so that only 1/8 of the photo diode current goes to the ADC. This mode is used in high-brightness situations to avoid saturation/clipping effects of the ADC. Default: `false`.
   | `continuous_mode: bool`                       | In default mode (`false`) the IC measures lux intensity only once every 800ms regardless of integration time. This mode allows the part to operate at its lowest possible supply current. <br><br> In continuous mode (`true`) the IC continuously measures lux intensity. That is, as soon as one reading is finished, a new one begins. If integration time is 6.25ms, readings are taken every 6.25ms. If integration time is 800ms, readings are taken every 800ms. In this mode, the part consumes slightly higher power than in the default mode. Default: `false`.

#### Modalities

Name                            | Description                           
--------------------------------|---------------------------------------
`intensity: uint32`             | The light intensity of the MAX44009.

### Noise Sensor

> The noise sensor uses the XDK's microphone to detect noise around it. The noise is calculated over 256 microphone samples. Note that it is unitless and needs to be calibrated/converted by your application if you require absolute instead of relative noise levels.

Reviewer: ```suggestion
The noise sensor uses the XDK's microphone to detect noise around it. The noise is calculated over 256 microphone samples. Note that this value without unit and needs to be calibrated/converted by your application if you require absolute noise level measurements.
```

   | `authentication: MqttAuthentication` | Username/Password authentication or unauthenticated, for example `authentication = Login("user", "pass")`. Default: `None()`.
   | `cleanSession: bool`                 | The clean session flag indicates to the broker whether the client wants to establish a clean session or a persistent session where all subscriptions and messages (QoS 1 & 2) are stored for the client. Default: `false`.
   | `keepAliveInterval: uint32`          | The keep alive interval (in seconds) is the time the client commits to for when sending regular pings to the broker. The broker responds to the pings enabling both sides to determine if the other one is still alive and reachable. Default: `60`.

#### Signals

Name                            | Description                           | Parameters | 
--------------------------------|---------------------------------------|------------|------------
`topic: string`  | Publishes a message to a particular topic.  | `name: string`  | The topic's name.
 || `qos: uint32`  | Default: `0`

### Eclipse Hono

> Mita implements a `HonoMqtt` resource you can use to easily connect to a [Hono](https://www.eclipse.org/hono/) network via MQTT.

Reviewer: ```suggestion
Mita implements a `HonoMqtt` resource you can use to easily connect to an [Eclipse Hono](https://www.eclipse.org/hono/) service via MQTT.
```
